packages/engine/Source/Scene/ScreenSpaceCameraController.js

```diff
@@ -591,9 +591,9 @@ function handleZoom(
     return;
   }
 
-  const sameStartPosition = Cartesian2.equals(
-    startPosition,
-    object._zoomMouseStart
+  const sameStartPosition = defaultValue(
```

Comment:
This looks like an unrelated change. Is it needed in this PR?

Reply:
I forgot to remove that, sorry